src/assets/stylesheets/variables.scss

```diff
@@ -47,4 +51,13 @@ $button-height-medium: 3.2rem;
 $button-height-small: 2.4rem;
 $button-border-width: 0.2rem;
 
+$page-padding-regular: 0 4.8rem;
+$page-padding-small: 0 2.4rem;
+
+$header-height-regular: 11.2rem;
+$header-height-small: 9.8rem;
+
+$fullscreen-height: 100vh;
+$fullscreen-width: 100vw;
```

Comment:
This seems a bit overkill, considering you could just use `100vw` or `100vh` 😄

Reply:
Well... it might be indeed 😅